lib/ae_mdw/aexn_contracts.ex

```diff
@@ -39,15 +39,52 @@ defmodule AeMdw.Contracts.AexnContract do
     end
   end
 
-  def is_aex141?(_no_fcode), do: false
+  @spec has_aex141_signatures?(pubkey()) :: boolean()
+  def has_aex141_signatures?(pubkey) do
+    case Contract.get_info(pubkey) do
+      {:ok, {type_info, _compiler_vsn, _source_hash}} ->
+        valid_aex141_signatures?(type_info)
+
+      {:error, _reason} ->
+        false
+    end
+  end
+
+  @spec has_valid_aex141_extensions?(Model.aexn_extensions(), pubkey() | Contract.type_info()) ::
+          boolean()
+  def has_valid_aex141_extensions?(extensions, pubkey) when is_binary(pubkey) do
+    case Contract.get_info(pubkey) do
+      {:ok, {type_info, _compiler_vsn, _source_hash}} ->
+        has_valid_aex141_extensions?(extensions, type_info)
+
+      {:error, _reason} ->
+        false
+    end
+  end
 
-  @spec call_meta_info(pubkey()) :: {:ok, Model.aexn_meta_info()} | :not_found
+  def has_valid_aex141_extensions?(extensions, {:fcode, functions, _hash_names, _code}) do
+    Enum.all?(extensions, &valid_aex141_extension?(&1, functions))
+  end
+
+  def has_valid_aex141_extensions?(_extensions, _no_fcode) do
```

Comment:
It's a handling for AEVM that also exists for AEX9. As it's legacy for the old VM, maybe the Node doesn't return the info in this other format anymore. It should be logged and checked to be safely removed.

One option would be to add `aevm_type :: any` in the spec but it would loosen the type check. 

For now I have just returned `false` as there would be no 141 at that time.